lib/core/test.js

```diff
@@ -86,7 +89,11 @@ Test.prototype.deployAll = function(contractsConfig, cb) {
       }
       self.web3.eth.defaultAccount = accounts[0];
       RunCode.doEval(result, self.web3); // jshint ignore:line
-      cb();
+      if (self.options.logging) {
+        cb(logger);
```

Comment:
I don't quite get why we're passing the logger object itself on the callback, a good alternative would be to do ```this.logger = ``` so the logger object is accessible in the tests if needed